components/src/core/ThreeLiner/ThreeLiner.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { HTMLAttributes } from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core';
+import clsx from 'clsx';
+
+export type ThreeLinerClasses = {
+    root?: string;
+    title?: string;
+    subtitle?: string;
+    info?: string;
+};
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            display: 'flex',
+            height: '100%',
+            flexDirection: 'column',
+            justifyContent: 'center',
```

Comment:
Just curious, does this need to be flex? It seems like it should work as block

Reply:
I used flex here so that the three liner would be center aligned when the toolbar is collapsed.  This is a little weird because the positioning here does not match angular.